src/webapi/xml_http_request.rs

```diff
@@ -91,6 +91,31 @@ impl XMLHttpRequest {
         };
     }
 
+    /// Returns the string containing the text of the specified header. If there
+    /// are multiple response headers with the same name, then their values are
+    /// returned as a single concatenated string.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getRequestHeader)
+    pub fn get_request_header(&self, header: &str) -> Option<String> {
+        let header = js!( return @{self}.getRequestHeader(@{header}); );
+        match header {
+            Value::Null => None,
+            Value::String(text) => Some(text),
+            _ => unreachable!(),
+        }
+    }
+
+    /// Sets the value of an HTTP request header. Must be called after `open()`,
+    /// but before `send()`. If this method is called several times with the same
+    /// header, the values are merged into one single request header.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/setRequestHeader)
+    pub fn set_request_header(&self, header: &str, value: &str) {
```

Comment:
Looking at the [specs](https://xhr.spec.whatwg.org/#the-setrequestheader()-method) this method can throw.

I guess we can leave it now as-is; one of the exceptions it can throw is the `InvalidStateError`, which coincidentially is being currently added in https://github.com/koute/stdweb/pull/64. (I guess we should do something with https://github.com/koute/stdweb/issues/28 rather sooner than later and have a more principled way of handling throwing methods.)

Please leave a `TODO` that this function throws and that we need to properly handle that.